spec/lib/mcb/commands/courses/show_spec.rb

```diff
@@ -9,11 +9,10 @@
   end
 
   it 'displays the course info' do
-    _site_status1 = create(:site_status)
+    subject2 = build(:subject)
+    site_status2 = build(:site_status)
```

Comment:
I think the only reason `2` was appended to these was to differentiate it from `_site_status1`. If we're going to get rid of that, which isn't a bad call, then may as well remove the `2`.